Makefile

```
@@ -578,6 +578,34 @@ kind-install-cilium-clustermesh: kind-clustermesh-ready ## Install a local Ciliu
 	$(CILIUM_CLI) clustermesh status --context kind-clustermesh1 --wait
 	$(CILIUM_CLI) clustermesh status --context kind-clustermesh2 --wait
 
+kind-install-cilium-clustermesh-fast:kind-clustermesh-ready
```

Missing space

```suggestion
kind-install-cilium-clustermesh-fast: kind-clustermesh-ready ## ...
```

It also misses the comment to briefly document the target so that the description shows up in `make help`.

More generally, should the `fast` target be documented somewhere?